scripts/next-release.sh

```diff
@@ -32,17 +38,31 @@ if ! command -v gh &>/dev/null; then
   exit 1
 fi
 
-LAST_MAJOR_MINOR_ZERO_RELEASE=$(gh release list --repo sourcegraph/sourcegraph --limit 20 --exclude-drafts --exclude-pre-releases | sed 's/Latest//' | awk '$3 ~ /v[0-9]+\.[0-9]+\.0$/ { print $3, $4; exit }')
-MAJOR_MINOR=$(echo $LAST_MAJOR_MINOR_ZERO_RELEASE | awk '{ print $1 }' | sed 's/v//' | cut -d. -f1,2)
+LAST_MAJOR_MINOR_ZERO_RELEASE=$(gh release list --repo sourcegraph/jetbrains --limit 20 --exclude-drafts --exclude-pre-releases | sed 's/Latest//' | awk '$2 ~ /v[0-9]+\.[0-9]+\.[0-9]+$/ { print $2, $3; exit }')
```

Comment:
the script logic is the same but we now depend on ours versioning instead of sourcegraph, is that right? plus we gain an ability to update major/minor from the script with the proper argument

Reply:
Yes, it's almost the same, regex is slightly different as sourcegraph had patch version always set to `.0` etc.
But no big changes.